src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -139,6 +147,9 @@ public void configure(Map<String, ?> configs) {
         quotaMap = config.getQuotaMap();
         long storageCheckInterval = config.getStorageCheckInterval();
         if (storageCheckInterval > 0L) {
+            LinkedHashMap<String, String> defaultTags = new LinkedHashMap<>();
+            defaultTags.put("observingBrokerId", config.getBrokerId());
```

Comment:
using the already defined constant here?

Reply:
Thanks fixed in a45646720e4e0a00d912a8bfa8bcc72df1db6558